src/mlpack/tests/ann_dist_test.cpp

```diff
@@ -127,4 +128,40 @@ BOOST_AUTO_TEST_CASE(JacobianBernoulliDistributionLogisticTest)
   }
 }
 
+/**
+ * Normal Distribution module test.
```

Comment:
Should we run the Jacobian tests for the normal distribution as well?

Reply:
Oh yes.. definitely! I've made the changes.. pls have a look